src/locales/locale-en.json

```diff
@@ -154,6 +155,11 @@
     "USERPROFILE": "Visit your profile",
     "CHANGE_GROUP": "Change group"
   },
+  "SIDENAV": {
+    "INFO_SUB": "How to build your foodsaving group",
```

Comment:
Does it make sense to have those translatable? The infopage is only available in English.

Reply:
I still think it makes sense to translate the text inside Karrot. And if only for consistency...